src/widgets/splits/SplitInput.hpp

```diff
@@ -100,7 +101,7 @@ class SplitInput : public BaseWidget
     /// Clears the input box, clears reply thread if inline replies are enabled
     void clearInput();
 
-protected:
+private:
```

Comment:
This was changed at one point during my work on #3722 but was never changed back. Made it's way through code review. I noticed it while working on this PR and figured I'd change it back.